### DISCUSSION TITLE
Javascript code templates http response debug

### DISCUSSION THREAD

**Discussion by KGorbakon** [projectdiscovery/nuclei#5642] [Q&A]
I ran into a problem. When writing templates with javascript code, the content of the http responses is not displayed entirely in debug mode (I tried such start options: -v, -svd, -debug, -dresp, -dreq). When starting nuclei with the -srd flag, the same result is repeated. When using burp as a proxy, requests are not intercepted. So how can I view the http responses entirely?

My template:
```  
javascript:
  - code: |   
      let m = require('nuclei/net');
      let address=Host+':'+Port;
      let conn;
      conn = m.OpenTLS('tcp', address)
      conn.Send('GET / HTTP/1.1\r\nHost:'+Host+'\r\nConnection: close\r\n\r\n');
      conn.RecvString();

    args:
      Host: "{{Host}}"
      Port: "443"
```

Debug info:
![image](https://github.com/user-attachments/assets/90fdad7a-6cb0-48eb-9651-ced02ff89fce)

Output stored in file:
![image](https://github.com/user-attachments/assets/6a125657-f6d4-4cf1-a6a0-adbaa2728df3)

Nuclei Engine Version: v3.3.1

**Comment by KGorbakon** [comment]
@ehsandeep @dwisiswant0 @tarunKoyalwar should i create issue from this discussion?

**Reply by GeorginaReeder** [reply]
Hey @KGorbakon - yes please, if you could go ahead and create an issue for this, that would be great.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.

**Comment by flyingllama87** [comment]
I'm running into the same issue. Did this get resolved?
